docroot/themes/contrib/civictheme/civictheme-library/components/03-organisms/listing/listing.scss

```diff
@@ -19,16 +36,22 @@
   }
 
   &.civictheme-theme-light {
+    &#{$root}__header {
```

Comment:
this is now `__title`

Reply:
I've refactored this to a cleaner approach to match card-container title and link